src/components/Story.svelte

```diff
@@ -1,39 +1,69 @@
-<script>
-  import { getStoryRenderContext, useContext } from './context';
+<script lang="ts" generics="M extends Meta">
+  import type { Meta, StoryObj, StoryContext } from '@storybook/svelte';
+  import type { Snippet } from 'svelte';
 
-  const context = useContext();
+  import { useStoriesExtractor, useStoryRenderer, useStoriesTemplate } from './context.svelte.js';
 
-  export let name;
-  export let template = null;
-  export let play = null;
+  type Props = {
+    children?: Snippet<[StoryObj<M>["args"], StoryContext<M["args"]>]>;
+    /**
+    * Id of the story.
+    *
+    * Optional, auto-generated from name if not specified.
+    */
+    id?: string;
+    /**
+    * Name of the story.
+    * @default "Default"
```

Comment:
I don't think making this optional with the "Default" name makes sense. This wasn't always the case right?

Reply:
Nope, it wasn't always the case. I've invented it to battle my own laziness while writing stories on my hobby project. I see this is opinionated and it indeed stands out. I'll remove it.